### PR TITLE
Convert parser tests to suite format, part 3

### PR DESCRIPTION
**Pull request by jbouwman**
Merge individual test cases into per-function files according to model established in #1274, #1278

- convert and, attribute, body, break, cond, continue, declare, let, toplevel

Several test cases which parse but do not compile are disabled pending review according to comments in #1272, #1274, #1278

### CODE REVIEW

File: tests/parser-test-files/parse-attribute.txt [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+(repr :hello)
+(define-type T)
+
+--------------------------------------------------------------------------------
+
+error: Unknown repr attribute
+  --> test:3:6
+   |
+ 3 |  (repr :hello)
```
Comment (Izaakwltn) [inline tests/parser-test-files/parse-attribute.txt:116]: `(repr :hello)` should be added, right? /s

**Review by Izaakwltn — APPROVED** [review]
One test change in parse-toplevel, otherwise good!

File: tests/parser-test-files/parse-toplevel.txt [hunk 2]
```diff
@@ -0,0 +1,40 @@
+================================================================================
+Parse toplevel (:disable)
+================================================================================
+
+(package coalton-unit-tests)
+
+(monomorphize)
```
[the remark this answers is not in the document]
Comment (jbouwman) [reply tests/parser-test-files/parse-toplevel.txt:7]: done

[the remark this answers is not in the document]
Comment (Izaakwltn) [reply tests/parser-test-files/parse-toplevel.txt:7]: you can take off `(:disable)` :)

[the remark this answers is not in the document]
Comment (jbouwman) [reply tests/parser-test-files/parse-toplevel.txt:7]: got it!

**Pull request merged**